java-api-core/src/com/kinvey/java/store/requests/user/GetMICTempURL.java

```diff
@@ -25,7 +25,7 @@
  * Created by Prots on 2/12/16.
  */
 public final class GetMICTempURL extends AbstractKinveyClientRequest<GenericJson> {
-    private static final String REST_PATH = "oauth/auth";
+    private static final String REST_PATH = "oauth/auth?scope=openid";
 
```

Comment:
Does this change have to do with the TLS fix?

Reply:
No, It's improvement for MIC. Do you think it should be filed as separate ticket(PR)?

Reply:
Yes I think that is best. I created `MLIBZ-2576` for this, please move this change to a branch for that ticket.